packages/fuels/src/cli/commands/deploy/proxy/contract/fuel-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+[toolchain]
+channel = "testnet-2024-09-06"
```

Comment:
Do we use the channel date here to fix the components at a known state?

Reply:
cc @Dhaiwat10

Reply:
That's a good question, maybe @K1-R1 can elaborate on it.

Nevertheless, I'm going with pre-built binaries so this TOML file will not exist anymore.

Reply:
correct @petertonysmith94 . Not strictly necessary; but has helped debugging efforts in the past.